<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produce_pencil</class>
 <widget class="QMainWindow" name="produce_pencil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>produce_pencil</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="PublicDemand">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Public Demand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PencilPrice">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Pencil Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pencil Producer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PencilProduced">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #696969;	
}</string>
    </property>
    <property name="text">
     <string>Pencils Produced (to date): </string>
    </property>
   </widget>
   <widget class="QLabel" name="textPencilProduced">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #696969;	
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="textPencilPrice">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>$1.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="textPublicDemand">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1.00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MakePencil">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #008000;	
	color: #FFFFFF;
	border: 0px;
}</string>
    </property>
    <property name="text">
     <string>Make Pencil</string>
    </property>
   </widget>
   <widget class="QLabel" name="PencilInventory">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Pencils in Inventory:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textPencilInventory">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChangePrice">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Change Price:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecreasePrice">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IncreasePrice">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="BankBalance_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Bank Balance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textBankBalance">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>$145,000.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>5</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>5</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Line {
	color: #000000;
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="WoodLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Wood Left</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GraphiteLeft">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Graphite Left</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textWoodLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1,000.00m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textGraphiteLeft">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1,000.00m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BuyWood">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #945213;	
	border: 0px;
	color: #FFFFFF;

}</string>
    </property>
    <property name="text">
     <string>Buy Wood</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BuyGraphite">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #696969;	
	border: 0px;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>Buy Graphite</string>
    </property>
   </widget>
   <widget class="QLabel" name="APM">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Automatic Pencil Machine 2000:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textAPM">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="APMRate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #696969;	
}</string>
    </property>
    <property name="text">
     <string>Rate: 120 pencils /  minute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BuyAPM">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e28800;	
	color: #FFFFFF;
	border: 0px;
}</string>
    </property>
    <property name="text">
     <string>Buy More APM</string>
    </property>
   </widget>
   <widget class="QLabel" name="textAPMPrice">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>$150.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textWoodPrice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>$50.00/100.00m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textGraphitePrice">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>$50.00/100.00m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
